rocAL/rocAL_pybind/examples/rocAL_api_coco_pipeline.py

```diff
@@ -45,13 +41,12 @@ class ROCALCOCOIterator(object):
            Epoch size.
     """
 
-    def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False, multiplier=None, offset=None, tensor_dtype=types.FLOAT, device="cpu", display=False, num_anchors=8732):
+    def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False, multiplier=None, offset=None, tensor_dtype=types.FLOAT, device="cpu", display=False, num_anchors=120087):
 
```

Comment:
why is the number of anchors changed from 8732. Probably we need to have 2 different pipelines (one for SSD and one for RetinaNet)

Reply:
This example shows the coco pipeline for RetinaNet.
Will add one for SSD pipeline as well in future PRs.

Reply:
please rename pipeline to "coco_pipeline_retinanet" and create another one for ssd (coco_pipeline_ssd)